src/Components/Artwork/SaveButton/useSaveArtwork.ts

```diff
@@ -40,6 +40,20 @@ export const useSaveArtwork = ({
                 slug: artwork.slug!,
                 is_saved: !isSaved,
               },
+              /**
+               * TODO: We don't _really_ need an optimistic response and
+               * it's too painful to alter all the consumers of this to pass
+               * in the `me` object.
+               *
+               * This hook shouldn't have to accept a stub `artwork` object either.
+               * Given just the artwork `id` we could fetch everything else.
+               */
+              me: {
+                id: "me",
+                counts: {
+                  savedArtworks: 0,
+                },
+              },
```

Comment:
What this means in practice is that there's a slight delay before the tip disappears; let's see how it feels in Q/A. I can fix this it's just a bit of a pain considering the state of these components.